sdk/core/Azure.Core/src/Serialization/SerializationExtensions.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+
+namespace Azure.Core.Serialization
```

Comment:
Is this the right namespace? We usually put stuff that we expect customers to use directly into Azure.

Reply:
I think Azure root would be better.